org.eclipse.dawnsci.remotedataset.server/src/org/eclipse/dawnsci/remotedataset/server/utils/DataServerUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.eclipse.dawnsci.remotedataset.server.utils;
+
+import org.eclipse.dawnsci.analysis.api.io.IDataHolder;
+import org.eclipse.dawnsci.remotedataset.ServiceHolder;
+import org.eclipse.january.IMonitor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DataServerUtils {
+	
+	private static Logger logger = LoggerFactory.getLogger(DataServerUtils.class);
+
+	public static IDataHolder getDataHolderWithLogging(String path) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00112)